packages/app/src/game/module/board/overlays/readyOverlay.tsx

```diff
@@ -51,29 +26,57 @@ const ReadyOverlay: React.FunctionComponent = () => {
 		const id = state.game.playerInfo.opponentId;
 		return state.game.playerList.find(p => p.id === id);
 	});
+
 	const spectatingPlayer = useSelector<AppState, boolean>(state => state.game.spectating.id !== null);
 
-	if (spectatingPlayer || !opponentInfo || !inReadyPhase) {
+	if (!opponentInfo || !inReadyPhase || spectatingPlayer) {
 		return null;
 	}
+	const renderHealthbar = (current: number) => current.toString();
 
-	const playerPosition = getPosition(playerInfo, playerList);
-	const opponentPosition = getPosition(opponentInfo, playerList);
+	const returnTitleOrBuffer = (player) => {
+		if (player.profile.title !== null) {
+			return <PlayerTitle playerId={player.id} />;
+		}
+		return <div className="buffer" />;
+	};
 
 	return (
 		<BoardOverlay>
 			<div className="ready-overlay-content">
-				<h3>Now Playing {opponentInfo.name}</h3>
-				<ul className="h2h">
-					<li>Health: <span className="highlight">{playerInfo.health} </span>
-						vs. <span className="opponent">{opponentInfo.health}</span></li>
-					<li>Level: <span className="highlight">{playerInfo.level} </span>
-						vs. <span className="opponent">{opponentInfo.level}</span></li>
-					<li>Streak: <span className="highlight">{playerInfo.streakAmount} {getStreakType(playerInfo)} </span>
-						vs. <span className="opponent">{opponentInfo.streakAmount} {getStreakType(opponentInfo)}</span></li>
-					<li>Position: <span className="highlight">{playerPosition}{getPositionModifier(playerPosition)} </span>
-						vs. <span className="opponent">{opponentPosition}{getPositionModifier(opponentPosition)}</span></li>
-				</ul>
+				<p className="h2h-header">Now Playing:</p>
+				<div className="outer-profile-box">
+					<div className="inner-profile-box">
+						<div className="player-picture">
+							<PlayerPicture playerId={playerInfo.id} />
+						</div>
+						<div className="name-and-health">
+							<p className="player-name">{playerInfo.name}</p>
+							{returnTitleOrBuffer(playerInfo)}
+							<ProgressBar
+								className="healthbar player-health h2h"
+								current={playerInfo.health}
+								max={100}
+								renderContents={renderHealthbar}
+							/>
+						</div>
+						<div className="buffer" />
+						<div className="name-and-health right">
+							<p className="player-name right">{opponentInfo.name}</p>
+							{returnTitleOrBuffer(opponentInfo)}
```

Comment:
Nice job with this function 👍  prefer "spacer" over "buffer" though - a `Buffer` is a specific data type (in most programming languages) so best to avoid confusion